src/python/ensembl/core/models.py

```diff
@@ -1970,29 +1765,8 @@ class SeqRegionAttrib(Base):
     )
     value = Column(Text, primary_key=True, nullable=False, index=True)
     seq_region = relationship("SeqRegion", back_populates="seq_region_attrib")
-    attrib_type = relationship("AttribType", back_populates="seq_region_attrib")
-
-
-#  t_seq_region_attrib = Table(
-#      "seq_region_attrib",
-#      metadata,
-#      Column(
-#          "seq_region_id",
-#          ForeignKey("seq_region.seq_region_id"),
-#          nullable=False,
-#          index=True,
-#          server_default=text("'0'"),
-#      ),
-#      Column(
-#          "attrib_type_id",
-#          ForeignKey("attrib_type.attrib_type_id"),
-#          nullable=False,
-#          server_default=text("'0'"),
-#      ),
-#      Column("value", Text, nullable=False, index=True),
-#      Index("type_val_idx", "attrib_type_id", "value"),
-#      Index("region_attribx", "seq_region_id", "attrib_type_id", "value", unique=True),
-#  )
+    attrib_type = relationship("AttribType", back_populates="seq_region_attributes")
```

Comment:
This needs to be reverted as well:
```suggestion
    attrib_type = relationship("AttribType", back_populates="seq_region_attrib")
```